kokoro/gcp_ubuntu/kokoro_build.sh

```diff
@@ -3,6 +3,15 @@
 # Fail on any error.
 set -e
 
+#Variables
+GCP_PROJECT         =$(gcloud config get-value project)
```

Comment:
AFAIK there must not be a space between the variable and the equals sign. I'm also not a fan of tabular layout as it requires a lot of reformatting when a line is changed.